platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -551,7 +551,7 @@ + (instancetype)expressionWithMGLJSONObject:(id)object {
                 return [NSExpression expressionForAggregate:MGLSubexpressionsWithJSONObjects(argumentObjects.firstObject)];
             }
             return [NSExpression expressionWithMGLJSONObject:argumentObjects.firstObject];
-        } else if ([op isEqualToString:@"to-boolean"]) {
+        } else if ([op isEqualToString:@"to-boolean"] || [op isEqualToString:@"boolean"]) {
```

Comment:
`boolValue` was only ever meant to coerce a value to a Boolean, not assert that it is a Boolean. Previously, `to-boolean` would be translated to `MGL_FUNCTION('to-boolean', …)`. Is there any potential for data loss this way?

Reply:
This was not meant to make it here. Removed